index.html

```diff
@@ -14,119 +14,207 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap"
       rel="stylesheet"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;700;800&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
     <meta name="theme-color" content="#e30062" />
   </head>
-
   <body>
-    <header>
-      <nav>
-        <!-- in case of small screens, show a dropdown -->
-        <input type="checkbox" id="nav-toggle" aria-controls="navbar toggle" />
-        <label for="nav-toggle" class="burger-menu"> &#9776;</label>
+    <div class="first-screen">
+      <header>
+        <a
+          href="https://www.realdevsquad.com/"
+          target="_blank"
+          class="logo-link"
+        >
+          <img
+            id="logo"
+            src="../img\re-designed images\RDS-logo.png"
+            alt="logo"
+          />
+        </a>
 
-        <ul class="menu">
-          <li>
-            <a class="nav-logo" href="https://realdevsquad.com/"
-              ><img
-                class="nav-logo-img"
-                src="./img/logos/Real-Dev-Squad@1x.png"
-                alt="Real Dev Squad logo"
-            /></a>
-            <a class="nav-home" href="https://realdevsquad.com/">Home</a>
-          </li>
-          <li>
-            <a class="active" href="https://welcome.realdevsquad.com/"
-              >Welcome</a
-            >
-          </li>
-          <li><a href="https://www.realdevsquad.com/events.html">Events</a></li>
-          <li><a href="https://members.realdevsquad.com/">Members</a></li>
-          <li><a href="https://crypto.realdevsquad.com/">Crypto</a></li>
-          <li><a href="https://status.realdevsquad.com/">Status</a></li>
-        </ul>
+        <div class="hamburger">
+          <div class="hamburger-lines"></div>
+          <div class="hamburger-lines"></div>
+          <div class="hamburger-lines"></div>
+        </div>
 
-        <div class="login-btn">
+        <nav class="main-nav">
           <a
-            href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+            class="nav-text welcome"
```

Comment:
are we having two different classes `nav-text` and `welcome` on this element?

Reply:
yes, actually that option needed to be styled differently @swarajpure